app/components/project-card-members.js

```diff
@@ -2,20 +2,59 @@ import Ember from 'ember';
 
 const VISIBLE_MEMBERS_COUNT = 8;
 
+/**
+  `project-card-members' displays the first 8 members of a project and some statistics,
```

Comment:
I think there is an apostrophe around `project-card-members` that should be a back tick